opentargets_pharmgkb/evidence_generation.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import pandas as pd
+
+from opentargets_pharmgkb.variant_coordinates import get_coordinates_for_clinical_annotation
+
+pgkb_column_names = [
+    'Clinical Annotation ID',
+    'Variant/Haplotypes',
+    'Gene',
+    'Level of Evidence',
+    'Phenotype Category',
+    'Drug(s)',
+    'Phenotype(s)',
+    'Genotype/Allele',
+    'Annotation Text'
+]
+
+
+def pipeline(clinical_annot_path, clinical_alleles_path, created_date, output_path):
+    clinical_annot_table = pd.read_csv(clinical_annot_path, sep='\t')
+    clinical_alleles_table = pd.read_csv(clinical_alleles_path, sep='\t')
+
+    merged_table = pd.merge(clinical_annot_table, clinical_alleles_table, on='Clinical Annotation ID')
+    rs_only_table = merged_table[merged_table['Variant/Haplotypes'].str.contains('rs')][pgkb_column_names]
+
+    # Also provide a column with all genotypes for a given rs
+    rs_only_table = pd.merge(rs_only_table, rs_only_table.groupby(by='Clinical Annotation ID').aggregate(
```

Comment:
Is there any indication of the failure to merge ? 
What do we do if a clinical annotation is not present in on of the table ?

Reply:
`merge` [defaults to an inner join](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.merge.html), so you're right we wouldn't get a warning if something was missing... I imagine we probably want `left` in this case as the alleles table seems less critical for completeness.